<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>AClient</class>
  <widget class="QMainWindow" name="AClient">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>450</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>640</width>
        <height>450</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>660</width>
        <height>450</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>发送方</string>
    </property>
    <property name="windowIcon">
      <iconset resource="../res.qrc">
        <normaloff>:/transfer/res/transfer/fileTrans.png</normaloff>
        :/transfer/res/transfer/fileTrans.png
      </iconset>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
          <widget class="QSplitter" name="splitter">
            <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <widget class="QWidget" name="layoutWidget">
              <layout class="QVBoxLayout" name="verticalLayout">
                <property name="topMargin">
                  <number>1</number>
                </property>
                <item>
                  <widget class="QLabel" name="label">
                    <property name="maximumSize">
                      <size>
                        <width>16777215</width>
                        <height>24</height>
                      </size>
                    </property>
                    <property name="font">
                      <font>
                        <pointsize>11</pointsize>
                        <bold>true</bold>
                      </font>
                    </property>
                    <property name="text">
                      <string>文件列表</string>
                    </property>
                    <property name="alignment">
                      <set>Qt::AlignmentFlag::AlignCenter</set>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QListWidget" name="FileListWidget">
                    <property name="maximumSize">
                      <size>
                        <width>350</width>
                        <height>16777215</height>
                      </size>
                    </property>
                    <property name="contextMenuPolicy">
                      <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
                    </property>
                    <property name="editTriggers">
                      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                    </property>
                    <property name="dragEnabled">
                      <bool>false</bool>
                    </property>
                    <property name="dragDropMode">
                      <enum>QAbstractItemView::DragDropMode::NoDragDrop</enum>
                    </property>
                    <property name="defaultDropAction">
                      <enum>Qt::DropAction::IgnoreAction</enum>
                    </property>
                    <property name="selectionMode">
                      <enum>
                        QAbstractItemView::SelectionMode::ExtendedSelection
                      </enum>
                    </property>
                    <property name="selectionBehavior">
                      <enum>QAbstractItemView::SelectionBehavior::SelectRows
                      </enum>
                    </property>
                  </widget>
                </item>
              </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget">
              <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                  <widget class="QLabel" name="label_2">
                    <property name="maximumSize">
                      <size>
                        <width>16777215</width>
                        <height>24</height>
                      </size>
                    </property>
                    <property name="font">
                      <font>
                        <pointsize>11</pointsize>
                        <bold>true</bold>
                      </font>
                    </property>
                    <property name="text">
                      <string>传输详情</string>
                    </property>
                    <property name="alignment">
                      <set>Qt::AlignmentFlag::AlignCenter</set>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QWidget" name="TransDetail" native="true">
                    <property name="minimumSize">
                      <size>
                        <width>250</width>
                        <height>0</height>
                      </size>
                    </property>
                    <property name="styleSheet">
                      <string notr="true">QWidget {
                        border: 1px solid;
                        }

                        QLabel {
                        border: none; /* 确保 QLabel 的边框不被修改 */
                        }
                      </string>
                    </property>
                    <layout class="QFormLayout" name="formLayout">
                      <item row="0" column="0">
                        <widget class="QLabel" name="lb1">
                          <property name="text">
                            <string>状态：</string>
                          </property>
                        </widget>
                      </item>
                      <item row="0" column="1">
                        <widget class="QLabel" name="lbStatus">
                          <property name="text">
                            <string>unknow</string>
                          </property>
                        </widget>
                      </item>
                      <item row="1" column="0">
                        <widget class="QLabel" name="lb2">
                          <property name="text">
                            <string>当前文件：</string>
                          </property>
                        </widget>
                      </item>
                      <item row="1" column="1">
                        <widget class="QLabel" name="lbCurFile">
                          <property name="text">
                            <string>unknow</string>
                          </property>
                        </widget>
                      </item>
                      <item row="2" column="0">
                        <widget class="QLabel" name="lb3">
                          <property name="text">
                            <string>大小：</string>
                          </property>
                        </widget>
                      </item>
                      <item row="2" column="1">
                        <widget class="QLabel" name="lbSize">
                          <property name="text">
                            <string>unknow</string>
                          </property>
                        </widget>
                      </item>
                      <item row="3" column="0">
                        <widget class="QLabel" name="lb4">
                          <property name="text">
                            <string>速度：</string>
                          </property>
                        </widget>
                      </item>
                      <item row="3" column="1">
                        <widget class="QLabel" name="lbSpeed">
                          <property name="text">
                            <string>unknow</string>
                          </property>
                        </widget>
                      </item>
                      <item row="4" column="0">
                        <widget class="QLabel" name="lb5">
                          <property name="text">
                            <string>预计剩余时间：</string>
                          </property>
                        </widget>
                      </item>
                      <item row="4" column="1">
                        <widget class="QLabel" name="lbRemaingTime">
                          <property name="text">
                            <string>unknow</string>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </widget>
                </item>
              </layout>
            </widget>
          </widget>
        </item>
      </layout>
    </widget>
    <widget class="QStatusBar" name="statusbar"/>
    <widget class="QToolBar" name="toolBar">
      <property name="windowTitle">
        <string>toolBar</string>
      </property>
      <property name="movable">
        <bool>false</bool>
      </property>
      <property name="allowedAreas">
        <set>Qt::ToolBarArea::TopToolBarArea</set>
      </property>
      <property name="iconSize">
        <size>
          <width>25</width>
          <height>25</height>
        </size>
      </property>
      <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
      </property>
      <attribute name="toolBarArea">
        <enum>TopToolBarArea</enum>
      </attribute>
      <attribute name="toolBarBreak">
        <bool>false</bool>
      </attribute>
      <addaction name="actionConnect"/>
      <addaction name="actionDisConnect"/>
      <addaction name="actionSend"/>
      <addaction name="actionAdd"/>
      <addaction name="actionHelp"/>
    </widget>
    <action name="actionSend">
      <property name="enabled">
        <bool>false</bool>
      </property>
      <property name="icon">
        <iconset resource="../res.qrc">
          <normaloff>:/transfer/res/transfer/tranStart.png</normaloff>
          :/transfer/res/transfer/tranStart.png
        </iconset>
      </property>
      <property name="text">
        <string>发送</string>
      </property>
      <property name="toolTip">
        <string>开始发送.</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="actionDisConnect">
      <property name="enabled">
        <bool>false</bool>
      </property>
      <property name="icon">
        <iconset resource="../res.qrc">
          <normaloff>:/transfer/res/transfer/disConnect.png</normaloff>
          :/transfer/res/transfer/disConnect.png
        </iconset>
      </property>
      <property name="text">
        <string>断开</string>
      </property>
      <property name="toolTip">
        <string>断开与接收方的连接.</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="actionHelp">
      <property name="icon">
        <iconset resource="../res.qrc">
          <normaloff>:/transfer/res/transfer/help.png</normaloff>
          :/transfer/res/transfer/help.png
        </iconset>
      </property>
      <property name="text">
        <string>帮助</string>
      </property>
      <property name="toolTip">
        <string>查看使用说明.</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="actionAdd">
      <property name="enabled">
        <bool>false</bool>
      </property>
      <property name="icon">
        <iconset resource="../res.qrc">
          <normaloff>:/transfer/res/transfer/addFile.png</normaloff>
          :/transfer/res/transfer/addFile.png
        </iconset>
      </property>
      <property name="text">
        <string>添加文件</string>
      </property>
      <property name="toolTip">
        <string>添加文件到列表.</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="actionRemove">
      <property name="icon">
        <iconset resource="../res.qrc">
          <normaloff>:/transfer/res/transfer/removeFileFromList.png</normaloff>
          :/transfer/res/transfer/removeFileFromList.png
        </iconset>
      </property>
      <property name="text">
        <string>移除这条</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="actionClearList">
      <property name="icon">
        <iconset resource="../res.qrc">
          <normaloff>:/transfer/res/transfer/clearList.png</normaloff>
          :/transfer/res/transfer/clearList.png
        </iconset>
      </property>
      <property name="text">
        <string>清空列表</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
    <action name="actionConnect">
      <property name="icon">
        <iconset resource="../res.qrc">
          <normaloff>:/transfer/res/transfer/Connect.png</normaloff>
          :/transfer/res/transfer/Connect.png
        </iconset>
      </property>
      <property name="text">
        <string>连接</string>
      </property>
      <property name="toolTip">
        <string>连接到接收方.</string>
      </property>
      <property name="menuRole">
        <enum>QAction::MenuRole::NoRole</enum>
      </property>
    </action>
  </widget>
  <resources>
    <include location="../res.qrc"/>
  </resources>
  <connections/>
</ui>